crisischeckin/CrisisCheckinMobile/CrisisCheckinMobile/LoginPage.cs

```diff
@@ -16,8 +17,9 @@ public class LoginPage : ContentPage
             if (isValid)
             {
                 //App.IsUserLoggedIn = true;
-                Navigation.InsertPageBefore(new ProfileView(), this);
-                await Navigation.PopAsync();
+                //Navigation.InsertPageBefore(new ProfileView(), this);
+                //await Navigation.PopAsync();
+                await Navigation.PushModalAsync(new Views.MainTabbedView(), true);
```

Comment:
yeah so the reason this was doing `InsertPageBefore` was as part of preventing the login page to be part of the navigation stack We don't want to keep going back until we end up at the login page again, if we're already signed in.